workshop/content/030_basic_content/075_serverless/_index.md

```diff
@@ -5,7 +5,264 @@ weight: 75
+
+* use the `FisWorkshopLambdaFailure` template
+* add a `Name` tag of `FisWorkshopLambdaFailure1`
+* confirm that you want to start the experiment
+* ensure that the "State" is `Running`
+
+{{% expand "Troubleshooting" %}}
+If the experiment fails to run, review the **Troubleshooting** heading at the bottom of the [**FIS SSM Start Automation Setup**]({{< ref "030_basic_content/030_basic_experiment/" >}}) section. Note that there is a possibility that SSM automation will not start at all. If this is the case, especially if you have manually created any of the resources, double check that you are specifying full ARNs rather than just names in the FIS template.
+{{% /expand %}}
+
+
+In the FIS window select the "Timeline" tab and hit refresh every minute or so. You should see the experiment progressing through the individual states with green indicating finished steps, blue indicating in-progress steps, and grey signifying steps yet to be started:
+
+{{< img "timeline.png" "Experiment timeline" >}}
+
+At the same time, watch the curl output in the CloudShell window. As the experiment transitions from one step to the next you should see the output change, first to increase the latency:
+
+```
+...
+Hello from Lambda! - 200 - 0.541716
+Hello from Lambda! - 200 - 0.513623
+Hello from Lambda! - 200 - 0.546924
+...
+```
+
+then for the latency to return to normal but the response code changing to 404:
+
+```
+...
+Hello from Lambda! - 404 - 0.113380
+Hello from Lambda! - 404 - 0.274236
+Hello from Lambda! - 404 - 0.136305
+...
+```
+
+and finally changing to an error message indicating an exception has occured during code execution:
+
+```
+...
+{"message": "Internal server error"} - 502 - 0.215665
+{"message": "Internal server error"} - 502 - 0.113820
+{"message": "Internal server error"} - 502 - 0.163391
+...
+```
+
+before returning to normal at the end of the experiment.
+
+Congratulations for completing this lab! In this lab you walked through running a multi-step experiment changed an SSM Parameter Store parameter and injected faults into a Lambda function. 
+
+
+## Learning and improving
+
+The setup we've shown here provides failure modes similar to those availabel for instances and containers. It also has various problems that you can experiment with and use for ideation on how to customize your own serverless fault injection libraries:
+
+* If you stop the FIS experiment prematurely, the parameter will not be reset to a non-impacting configuration. To address this you could add a `NoFaultParameterValue` parameter to the SSM document / FIS template and add an `onError` / `onCancel` path to the SSM document. You could even read the parameter at the start of the SSM document run, but consider concurrency implications if anyother experiment is also changing the parameter.
```

Comment:
Why don't you add it already?  

https://github.com/adhorn/aws-fis-templates-cdk/blob/main/lib/fis-upload-ssm-docs/documents/ssma-put-config-parameterstore.yml#L87

Reply:
Hmm ... this was on purpose ... I am teaching them how to fish and this is a very important thing for them to consider when building their own. I will update the text to clarify why and link to your repo.